test/tbb/test_partitioner.cpp

```diff
@@ -28,6 +29,7 @@
 #include <cstddef>
 #include <utility>
 #include <vector>
+#include <algorithm>
```

Comment:
When including "big" headers with lots of functions of which you only need a couple, it makes sense to say in a comment which function(s) are used.

Reply:
done